docs/unsat_core_extraction.md

```diff
@@ -2,7 +2,7 @@
 
 When a model is unsatisfiable, it can be desirable to get a better idea of which Boolean variables make it unsatisfiable. Commonly, these Boolean variables are 'switches' that turn constraints on, hence such Boolean variables can be used to get a better idea of which _constraints_ make the model unsatisfiable.
 
-In the SATisfiability literature, the Boolean variables of interests are called _assumption_ variables and the solver will assume they are true. The subset of these variables that, when true, make the model unsatisfiable is called an unsatisfiable _core_.
+In the satisfiability literature, the Boolean variables of interests are called _assumption variables_ and the solver will assume they are true. The subset of these variables that, when true, makes the model unsatisfiable is called an unsatisfiable _core_.
```

Comment:
Technically, they are literals right?

Reply:
Yes, I guess so, especially since we're saying the assumptions are "true", we should call them literals. IPASIR2 also calls them assumption literals.

Reply:
I've changed it now so that this section is technically correct (and I think we should call them "assumptions" so they can be literals or even e.g. integers in the future). Let me know if it's worth it to change this throughout the rest of the doc/code.

Reply:
as you wish